### DISCUSSION TITLE
What do the colors for event markers mean?

### DISCUSSION THREAD

**Discussion by KillerTheFlareon** [TEHE-Studios/ExpandedHelicopterEvents#120] [Q&A] [answered]
I noticed that helicopter events seem to have colors now? The last two times a heli came within range I saw it had a green tint.
What do they mean, is there multiple colors and uh..how can you tell if a heli is just gonna start unloading with you? Every other encounter I have with one appears to attempt to attack me.

Admittedly this probably stops good events like airdrops from..well, having a airdrop since I just hide inside whenever I get the slightest hint of a helicopter showing up.

**Comment by Chuckleberry-Finn** [comment, marked as answer]
Please keep question titles accurate.

## Non-Aggressive Military: Green | Aggressive Military and raiders: Red | Police: Blue | Civilian/Default: Grey | Weird: Purple
